physics/cu_gf_driver.F90

```diff
@@ -7,7 +7,7 @@ module cu_gf_driver
    ! DH* TODO: replace constants with arguments to cu_gf_driver_run
    !use physcons  , g => con_g, cp => con_cp, xlv => con_hvap, r_v => con_rv
    use machine   , only: kind_phys
-   use cu_gf_deep, only: cu_gf_deep_run,neg_check,fct1d3
+   use cu_gf_deep, only: cu_gf_deep_run,neg_check,autoconv,aeroevap,fct1d3
```

Comment:
autoconv and aeroevap should no longer be in this use statement

Reply:
I believe this was a workaround for a compiler issue. The compiler wanted some extra things in the scope that it shouldn't need.

Reply:
Strange, we did not run into that issue before. But I guess it is not a big deal.

Reply:
Remember that the community repository is more widely tested than our branches. There are many compiler versions on many platforms, each of which has its own eccentricities.